python/tank/api.py

```diff
@@ -592,10 +594,21 @@ def preview_filesystem_structure(self, entity_type, entity_id, engine=None):
                                                       engine)
         return folders
 
+    @property
+    def local_site_cache_location(self):
+        """
+        Returns the location of the site cache root for this Tank instance.
+        """
+        # Invoke the global method which is generic and can compute any site
+        # cache root.
+        return path.get_local_site_cache_location(self.shotgun.base_url)
```

Comment:
See above comments - I think it would be grand if this code also ensures that this folder exists on disk, with the correct permissions etc.